packages/platforms/accton/x86-64/wedge100bf-32x/onlp/builds/x86_64_accton_wedge100bf_32x/module/src/platform_lib.c

```diff
@@ -91,6 +92,16 @@ static int tty_exec_buf(unsigned long udelay, const char *str)
     return (strstr(tty_buf, str) != NULL) ? 0 : -1;
 }
 
+/*Clear Rx buffer by reading it out.*/
```

Comment:
Please follow the Linux kernel coding style, and add a space after `/*` and before `*/`.